x-pack/plugins/observability/public/pages/slos/components/badges/slo_badges.tsx

```diff
@@ -24,45 +24,64 @@ export interface Props {
   rules: Array<Rule<SloRule>> | undefined;
   slo: SLOWithSummaryResponse;
   onClickRuleBadge: () => void;
+  cardView?: boolean;
 }
 
-export function SloBadges({ activeAlerts, isLoading, rules, slo, onClickRuleBadge }: Props) {
+export function SloBadges({
+  activeAlerts,
+  isLoading,
+  rules,
+  slo,
+  onClickRuleBadge,
+  cardView,
+}: Props) {
   return (
-    <EuiFlexGroup direction="row" responsive={false} gutterSize="s" alignItems="center" wrap>
-      {isLoading ? (
-        <>
-          <EuiSkeletonRectangle
-            isLoading
-            contentAriaLabel="Loading"
-            width="54.16px"
-            height="20px"
-            borderRadius="s"
-          />
-          <EuiSkeletonRectangle
-            isLoading
-            contentAriaLabel="Loading"
-            width="54.16px"
-            height="20px"
-            borderRadius="s"
-          />
-          <EuiSkeletonRectangle
-            isLoading
-            contentAriaLabel="Loading"
-            width="54.16px"
-            height="20px"
-            borderRadius="s"
-          />
-        </>
-      ) : (
+    <>
+      <EuiFlexGroup direction="row" responsive={false} gutterSize="s" alignItems="center" wrap>
+        {isLoading ? (
+          <>
+            <EuiSkeletonRectangle
+              isLoading
+              contentAriaLabel="Loading"
+              width="54.16px"
+              height="20px"
+              borderRadius="s"
+            />
+            <EuiSkeletonRectangle
+              isLoading
+              contentAriaLabel="Loading"
+              width="54.16px"
+              height="20px"
+              borderRadius="s"
+            />
+            <EuiSkeletonRectangle
+              isLoading
+              contentAriaLabel="Loading"
+              width="54.16px"
+              height="20px"
+              borderRadius="s"
+            />
+          </>
+        ) : (
+          <>
+            {!cardView && <SloStatusBadge slo={slo} />}
+            {!cardView && <SloGroupByBadge slo={slo} cardView={cardView} />}
```

Comment:
This relates to my comment [above](https://github.com/elastic/kibana/pull/171422/files#r1400197239). Since you are not showing the group by badge in the compact view, you don't need to pass it at all as a prop. Do I miss something, here?